source/_integrations/ituran.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: Ituran
+description: Instructions on how to add Ituran to Home Assistant.
+ha_category:
+  - Car
+  - Device Tracker
+  - Presence detection
+ha_release: '2025.1'
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@shmuelzon'
+ha_domain: ituran
+ha_platforms:
+  - device_tracker
+ha_integration_type: integration
+---
+
+The **Ituran** {% term integration %} allows you to retrieve information from your Ituran-equipped vehicle. It pulls information from the Ituran web service regarding the vehicle's location.
+
+## Prerequisites
+
+You must have an Ituran account for use with the Ituran APP ([Android](https://play.google.com/store/apps/details?id=com.ituran.forall)/[iOS](https://apps.apple.com/app/id1227756834)). All devices that work with the app, should work with this integration as well.
+
+## Data updates
+
+The information is pulled every 5 minutes from the Ituran web service; however, the data is updated at intervals determined by Ituran based on whether the vehicle is stationary or not.
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+ID or passport number:
+  description: Your government ID or passport number used to sign-up with Ituran.
+Mobile phone number:
+  description: The mobile phone number used to sign-up with Ituran. A one-time-password will be sent to this number.
+{% endconfiguration_basic %}
+
+## Platforms
+
+### Device tracker
+
+The Ituran {% term integration %} will track the location of each vehicle registered to your account.
+
+## Known limitations
+
+- While this integration is configured with your account, you won't be able to use the official app, as only one connection at a time is supported
+
+{% include integrations/remove_device_service.md %}
```

Comment:
Let's use the standard as put here: https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-removal-instructions

Reply:
I took the standard but didn't include the last sentence:
> After deleting the integration, go to the app of the manufacturer and remove the Home Assistant integration from there as well.

As I don't think it's relevant here.